lib/srv/desktop/windows_server.go

```diff
@@ -455,7 +455,7 @@ func (s *WindowsService) tlsConfigForLDAP() (*tls.Config, error) {
 		using to sign in. This is set to become a strict requirement by May 2023,
 		please update your configuration file before then.`)
 	}
-	certDER, keyDER, err := s.generateCredentials(s.closeCtx, user, s.cfg.Domain, windowsDesktopServiceCertTTL, s.cfg.SID)
+	certDER, keyDER, err := s.generateCredentials(s.closeCtx, user, s.cfg.Domain, windowsDesktopServiceCertTTL, s.cfg.SID, false, nil)
```

Comment:
There's enough arguments here that I'd consider making a struct.

```suggestion
	certDER, keyDER, err := s.generateCredentials(
		s.closeCtx, 
		generateCredentialsRequest{
			User: user,
			Domain: s.cfg.Domain,
			TTL: windowsDesktopServiceCertTTL,
			SID: s.cfg.SID,
			...
```

Reply:
That makes sense, I've changed it